.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '*--*'
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npm install -g @bazel/bazelisk
+      - run: cat .bazelrc.travis >> .bazelrc
```

Comment:
Why can't we move `.bazelrc.travis` to `.bazelrc`?

Reply:
We probably could.. that config's not used locally atm but we could probably change that.